### PR TITLE
experiment: use transient-lib

### PR DESCRIPTION
**Pull request by eugbyte**
1. Use transient storage lib to replace the native transient storage api implementation
2. Improve logging to log only relevant output from api-notarise-healthcert, and the logs from the transient-storage-lib

### CODE REVIEW

File: src/common/logger.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import debug from "debug";
 
-if (!process.env.JEST_WORKER_ID) debug.enable("*"); // enable log outputs if not running in jest
+debug.enable("api-notarize-healthcerts:*,transient-storage-lib:*");
```
Comment (HJunyuan) [inline src/common/logger.ts:3]: Btw is this equivalent to setting `DEBUG=api-notarize-healthcerts:*` in package.json?

File: package.json [hunk 2]
```diff
@@ -34,6 +34,8 @@
     "@notarise-gov-sg/eu-dcc-generator": "^1.1.1",
     "@notarise-gov-sg/gpay-covid-cards": "^1.0.5",
     "@notarise-gov-sg/sns-notify-recipients": "^1.5.2",
+    "@notarise-gov-sg/transient-storage-lib": "^0.3.1",
+    "@pathcheck/dcc-sdk": "0.0.17",
```
Comment (zawmyolatt) [inline package.json:38]: plz merge master update codes, this plugin is already removed.

File: src/common/logger.ts (hunk 1 above)
Comment (zawmyolatt) [inline src/common/logger.ts:3]: we also have other packages log like gpay, eu-dcc packages etc. so, can u keep it as `*` like before ?

**Review by HJunyuan — APPROVED** [review]

File: src/common/logger.ts (hunk 1 above)
Comment (eugbyte) [reply src/common/logger.ts:3]: I will revert the changes

**Review by zawmyolatt — APPROVED** [review]

**Pull request merged**